packages/manager/cypress/e2e/core/oneClickApps/one-click-apps.spec.ts

```diff
@@ -34,17 +34,22 @@ describe('OneClick Apps (OCA)', () => {
         // For every Marketplace app defined in Cloud Manager, make sure the API returns
         // the nessesary StackScript and that the app renders on the page.
         for (const stackscriptId in oneClickApps) {
-          const stackscript = stackScripts.find((s) => s.id === +stackscriptId);
-          const app = oneClickApps[stackscriptId];
+          const stackscript = stackScripts.find(
+            (stackScript) => stackScript.id === +stackscriptId
+          );
+
+          const displayLabel = stackscript
+            ? getMarketplaceAppLabel(stackscript.label)
+            : '';
```

Comment:
Can we move this below the `if` statement? No need to fallback to an empty string

```suggestion
          const displayLabel = getMarketplaceAppLabel(stackscript.label);
```

Reply:
Moving this line above the `if` statement would lead to a scope error for `displayLabel` as it is used in the error message that we throw on Lines 46-47.

You have to scroll far into the right to see it:
`We expected that StackScript to be in the response for the Marketplace app named "${displayLabel}".`

Reply:
Right, but wouldn't `displayLabel` be an empty string?

Reply:
We could change the error message to:
`Cloud Manager's fetch to GET /v4/linode/stackscripts did not receive a StackScript with ID ${stackscriptId}. We expected a StackScript to be in the response.`
This would allow us to make the change you suggested, resolve the empty string, and the deal with the variable scope error.

Reply:
Nice. Looks great now!